components/dependencies.mdx

```diff
@@ -1,59 +1,105 @@
 ---
-title: "Dependencies"
-description:
-  "Architect was designed with the future in mind, and the future of any team
-  building distributed software is a complex web of services."
+title: Dependencies
+description: Integrate with other APIs using common dependency management tactics
 ---
 
-Historically this complex web of services has been hard to manage, hard to
-reason about, and hard for developers to contribute to.
+Historically cloud teams have tried to avoid dependencies and coupling between microservices and APIs
+because it creates a lot of infrastructure and CI/CD complexity. However, its all too natural for
+developers to try to extend what exists rather than reinvent it, and many teams ended up with complex
+webs of microservices nonetheless. 
 
-Fortunately, the graphing and collaboration problems this creates have been
-solved before. We've seen it with object-oriented programming and inheritance,
-and even more recently with package and dependency management. All of our
-favorite languages have a way for developers to utilize dependencies through a
-resolver to handle the artifact storage, complex graphing, and de-duplication
-needed to manage them at-scale. Architect takes this same approach to make it
-easier than ever for developers to extend cloud services.
+Fortunately, this problem of "dependencies" has been solved before. Every language and operating system
+on earth has tools for dependency management to help developers collaborate and extend each others work.
+Now Architect is here to provide the same value for APIs and microservices.
 
-## Utilizing dependencies
+## Why use dependencies?
 
-```yaml
+Citing dependencies as part of your component is not the most natural way for developers to integrate with
+software made by other teams, but its also the only way to achieve [on-demand environments](/environments/pull-request)
+for [distributed applications](/architecture/microservices).
+
+Every time a component is deployed with Architect, it will automatically deploy (if needed) and connect to
+cited dependencies. Perhaps even more impressive than that is that it will also deploy the dependencies OF your
```

Comment:
```suggestion
cited dependencies. Perhaps even more impressive than that is that it will also deploy the *dependencies your
```

Reply:
I'm not sure this suggestion captures what I was trying to say which is that we deploy the dependencies you didn't know you had as well (e.g. the dependencies OF your own dependencies). Is there a better way to phrase that that you can think of?